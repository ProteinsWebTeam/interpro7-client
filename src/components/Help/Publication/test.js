import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import { PrintedPublication } from '.';

const renderer = new ShallowRenderer();

describe('<Publication />', () => {
  test('should render', () => {
    renderer.render(<PrintedPublication />);
    expect(renderer.getRenderOutput()).toMatchSnapshot();
  });
});
