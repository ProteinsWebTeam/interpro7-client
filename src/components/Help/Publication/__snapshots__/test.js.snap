// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Publication /> should render 1`] = `
<React.Fragment>
  <blockquote
    className="vf-blockquote"
  >
    <div />
    <div>
      <b />
    </div>
    <React.Fragment>
      <i />
      . 
      ,
       
      <Memo(Connect(Link))
        className="ext"
        href="https://doi.org/undefined"
        target="_blank"
      >
        doi: 
      </Memo(Connect(Link))>
    </React.Fragment>
  </blockquote>
</React.Fragment>
`;
